sqls/postgresql/restore/restore.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+# Create database credentials file
+cat > ~/.pgpass << EOF
+postgresql:${DB_PORT:-5432}:postgres:postgres:password
```

Comment:
Shouldn't this be configurable?

Reply:
This is actually not used, I'll delete it